lib/process-services-cloud/src/lib/rich-text-editor/editorjs-config.ts

```diff
@@ -86,7 +86,8 @@ export const editorJsConfig = {
         'Increase/Decrease font size': {
             class: ChangeFontSize,
             config: {
-                cssClass: 'plus20pc'
+                cssClass: 'plus20pc',
+                buttonIcon: '<span class="material-icons">format_size</span>'
```

Comment:
Does this not work?
```suggestion
                buttonIcon: '<mat-icon>format_size</mat-icon>'
```

Reply:
No, it doesn't
![image](https://github.com/user-attachments/assets/9fc4a728-3ba3-42cd-894f-d10bd63df52c)